### DISCUSSION TITLE
404 not found webpack Module

### DISCUSSION THREAD

**Discussion by Thunderfu** [vercel/next.js#17753] [Help] [answered]
I'm using the [swf-loader](https://github.com/openfl/swf-loader) webpack module on NextJS

However, it appears that the webpack is not creating the folder in the public folder **but it is creating in the .next folder**

next.config.js:
```
module.exports = (nextConfig = {}) => {
  return Object.assign({}, nextConfig, {
    webpack(config, options) {
      const { isServer } = options;

      if (!options.defaultLoaders) {
        throw new Error(
          "This plugin is not compatible with Next.js versions below 5.0.0 https://err.sh/next-plugins/upgrade"
        );
      }

      const prefix = nextConfig.assetPrefix || "";

      config.module.rules.push({
        test: /\.swf$/,
        use: [
          {
            loader: require.resolve("swf-loader")
          },
        ],
      });

      if (typeof nextConfig.webpack === "function") {
        return nextConfig.webpack(config, options);
      }

      return config;
    },
  });
};
```
I have 404 not found error in this url: **http://localhost:3000/_next/8fde4a2ad2ff7c98468bc611b5733844.bundle/library.json**

It seems that the folder was not created, however in .next folder:
![image](https://user-images.githubusercontent.com/72291584/95613721-c5a1de00-0a3b-11eb-8dba-868f19df60f1.png)

And inside the bundle folder:
![image](https://user-images.githubusercontent.com/72291584/95613663-ac009680-0a3b-11eb-861e-a8da5cf953d9.png)

**But I need to access this folder publicly**. And I think it was to have created in the public folder too

## System information

- Version of Next.js: [9.5.3]
- Version of Node.js: [12.18.4]

**Comment by jamesmosier** [comment, marked as answer]
Next places all build files from webpack in the `.next` folder, therefore you would have to manually move this file from there to the `public` folder. There is no built-in way to put files in the `public` folder.

The `next-offline` package [does something similar](https://github.com/hanford/next-offline/blob/master/packages/next-offline/export.js) by copying the file from `.next` to `public`.

**Reply by Thunderfu** [reply]
But there is some way for me to access these files created by the webpack via the URL. Or is the only way to move manually?

**Reply by jamesmosier** [reply]
Unfortunately no, there is no way to programmatically retrieve these and display/make them available to the user and have them be public. The only way is to put them in the `public` folder during your build.
